dropwizard-jersey/src/main/java/io/dropwizard/jersey/jackson/JsonProcessingExceptionMapper.java

```diff
@@ -17,7 +18,7 @@
 import java.util.regex.Pattern;
 
 @Provider
-public class JsonProcessingExceptionMapper implements ExceptionMapper<JsonProcessingException> {
+public class JsonProcessingExceptionMapper extends LoggingExceptionMapper<JsonProcessingException> implements ExceptionMapper<JsonProcessingException> {
```

Comment:
`LoggingExceptionMapper` itself implements `ExceptionMapper`, so the implements clause here isn't necessary.